zuul.d/project.yaml

```diff
@@ -6,6 +6,10 @@
       jobs:
         - ansible-cloud-vcenter-appliance:
             voting: false
+        - ansible-security-integration-qradar-python38:
+            files:
+              - playbooks/ansible-test-network-integration-base/files/bootstrap-qradar.yaml
```

Comment:
playbooks/ansible-network-appliance-base/.*

we want to trigger a job run here, is we change the base playbooks

Reply:
Oky, thanks for the feedback @pabelanger.